src/test/java/com/coherentsolutions/training/aqa/java/web/nikonova/objects/AutoWishListTest.java

```diff
@@ -0,0 +1,86 @@
+package com.coherentsolutions.training.aqa.java.web.nikonova.objects;
+
+import io.qameta.allure.Description;
+import io.qameta.allure.Feature;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class AutoWishListTest extends BaseTest {
+
+    @DataProvider
+    public Object[][] userLoginDetails() {
+        return new Object[][]{{"SeleniumTest789@yandex.com", "password123"}};
```

Comment:
please add more data sets for your DataProvider, because in this case you have only 1 data set and there is no reason to use DataProvider at all